<html xmlns:math="http://exslt.org/math" xmlns:testng="http://testng.org">
   <head>
      <title>TestNG Results</title>
      <meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
      <meta http-equiv="pragma" content="no-cache"></meta>
      <meta http-equiv="cache-control" content="max-age=0"></meta>
      <meta http-equiv="cache-control" content="no-cache"></meta>
      <meta http-equiv="cache-control" content="no-store"></meta>
      <LINK rel="stylesheet" href="style.css"></LINK>
      <script type="text/javascript" src="main.js"></script>
   </head>
   <body>
      <table width="100%" style="font-size: 16px; margin-bottom: 10px;" cellspacing="1">
         <tr>
            <td width="100%">
               Test case
               
               <b>Test</b>
            </td>
            <td style="background-color: #FFBBBB; padding: 3px 3px 3px 0;" align="center">
               <div style="width: 50px;">2</div>
            </td>
            <td style="background-color: lightgreen; padding: 3px 3px 3px 0;" align="center">
               <div style="width: 50px;">0</div>
            </td>
            <td style="background-color: #FFFFBB; padding: 3px 3px 3px 0;" align="center">
               <div style="width: 50px;">0</div>
            </td>
            <td align="center" style="background-color: #eaf0f7; padding: 3px 3px 3px 0;">
               <div style="width: 50px;">2</div>
            </td>
            <td align="center"
                style="font-weight: bold; background-color: #eaf0f7; padding: 3px 3px 3px 0;">
               <div style="width: 50px;">0%</div>
            </td>
            <td style="background-color: #eaf0f7; padding: 3px 3px 3px 0;" align="center"
                nowrap="true">7s
               
            </td>
         </tr>
      </table>
      <div style="width: 200px;">
         <label for="groupMethodsCheckBox" style="font-weight: bold; margin: 0;">
            <input id="groupMethodsCheckBox" type="checkbox" onclick="switchTestMethodsView(this)"></input>
            Group by class
            
         </label>
         <br></br>
         <label for="methodsFilter_ALL" style="font-weight: bold; margin: 0;">
            <input id="methodsFilter_ALL" type="checkbox"
                   onclick="testMethodsFilterChanged(this, 'ALL')"></input>
            All
            
         </label>
      </div>
      <label for="methodsFilter_FAIL" style="margin-left: 20px;">
         <input id="methodsFilter_FAIL" type="checkbox"
                onclick="testMethodsFilterChanged(this, 'FAIL')"
                checked=""></input>
         Failed
         
      </label>
      <label for="methodsFilter_PASS">
         <input id="methodsFilter_PASS" type="checkbox"
                onclick="testMethodsFilterChanged(this, 'PASS')"
                checked=""></input>
         Passed
         
      </label>
      <label for="methodsFilter_SKIP">
         <input id="methodsFilter_SKIP" type="checkbox"
                onclick="testMethodsFilterChanged(this, 'SKIP')"
                checked=""></input>
         Skipped
         
      </label>
      <label for="methodsFilter_CONF">
         <input id="methodsFilter_CONF" type="checkbox"
                onclick="testMethodsFilterChanged(this, 'CONF')"></input>
         Config
         
      </label>
      <br></br>
      <div id="testMethodsByStatus">
         <table class="testMethodsTable" cellpadding="0" cellspacing="0">
            <tr class="methodsTableHeader">
               <td width="100%">Name</td>
               <td nowrap="true">Started</td>
               <td nowrap="true">Duration</td>
               <td>Exception</td>
            </tr>
            <tr id="testCasePack.Req_12345_verifyURL_byStatus_failed_FAIL1_row"
                class="testMethodStatusFAIL">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_verifyURL_byStatus_failed_FAIL1_details')">verifyURL()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">14 ms
                  
               </td>
               <td nowrap="true">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_verifyURL_byStatus_failed_FAIL1_exception')">java.lang.Throwable: 
                     Multiple failures (2):
                     
                     Failure 1 of 2:
                     java.lang.AssertionError: expected [Googe] but found [Google]
                     	at org.testng.Assert.fail(Assert.java:94)
                     	at org.testng.Assert.failNotEquals(Assert.java:513)
                     	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                     	at org.testng.Assert.assertEquals(Assert.java:116)
                     	at org.testng.Assert.assertEquals(Assert.java:179)
                     	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                     	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                     	at testCasePack.Req_12345.verifyURL(Req_12345.java:27)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                     	at java.lang.reflect.Method.invoke(Method.java:497)
                     	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                     	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                     	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                     	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                     	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                     	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                     	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                     	at org.testng.TestRunner.run(TestRunner.java:607)
                     	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                     	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                     	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                     	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                     	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                     	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                     	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                     	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                     	at org.testng.TestNG.runSuites(TestNG.java:1124)
                     	at org.testng.TestNG.run(TestNG.java:1096)
                     	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                     	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                     	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                     	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                     	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                     	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     
                     
                     Failure 2 of 2:
                     java.lang.AssertionError: 'This is Message'------&gt; expected [true] but found [false]
                     
                  </a>
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_verifyURL_byStatus_failed_FAIL1_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>verifyURL
                     </div>
                     <div>
                        <b>Description: </b>Verify Title
                     </div>
                     <div>
                        <b>Signature: </b>verifyURL()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>14 ms
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_verifyURL_byStatus_failed_FAIL1_exception"
                       class="testMethodDetails">
                     <pre style="padding: 5px; margin: 0;">java.lang.Throwable: Multiple failures (2):
                        
                        Failure 1 of 2:
                        java.lang.AssertionError: expected [Googe] but found [Google]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                        	at org.testng.Assert.assertEquals(Assert.java:116)
                        	at org.testng.Assert.assertEquals(Assert.java:179)
                        	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                        	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                        	at testCasePack.Req_12345.verifyURL(Req_12345.java:27)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                        
                        
                        Failure 2 of 2:
                        java.lang.AssertionError: 'This is Message'------&gt; expected [true] but found [false]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertTrue(Assert.java:42)
                        	at utils.ErrorCollector.assertTrue(ErrorCollector.java:28)
                        	at utils.ErrorCollector.verifyTrue(ErrorCollector.java:65)
                        	at testCasePack.Req_12345.verifyURL(Req_12345.java:28)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     </pre>
                  </div>
               </td>
            </tr>
            <tr id="testCasePack.Req_12345_getCurrentURL_byStatus_failed_FAIL2_row"
                class="testMethodStatusFAIL">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_getCurrentURL_byStatus_failed_FAIL2_details')">getCurrentURL()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">8 ms
                  
               </td>
               <td nowrap="true">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_getCurrentURL_byStatus_failed_FAIL2_exception')">java.lang.Throwable: 
                     Multiple failures (2):
                     
                     Failure 1 of 2:
                     java.lang.AssertionError: expected [URL] but found [https://www.google.co.in/]
                     	at org.testng.Assert.fail(Assert.java:94)
                     	at org.testng.Assert.failNotEquals(Assert.java:513)
                     	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                     	at org.testng.Assert.assertEquals(Assert.java:116)
                     	at org.testng.Assert.assertEquals(Assert.java:179)
                     	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                     	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                     	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:35)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                     	at java.lang.reflect.Method.invoke(Method.java:497)
                     	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                     	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                     	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                     	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                     	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                     	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                     	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                     	at org.testng.TestRunner.run(TestRunner.java:607)
                     	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                     	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                     	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                     	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                     	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                     	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                     	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                     	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                     	at org.testng.TestNG.runSuites(TestNG.java:1124)
                     	at org.testng.TestNG.run(TestNG.java:1096)
                     	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                     	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                     	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                     	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                     	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                     	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     
                     
                     Failure 2 of 2:
                     java.lang.AssertionError: Message----&gt; expected [true] but found [false]
                     
                  </a>
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_getCurrentURL_byStatus_failed_FAIL2_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>getCurrentURL
                     </div>
                     <div>
                        <b>Description: </b>Verify URL
                     </div>
                     <div>
                        <b>Signature: </b>getCurrentURL()[pri:1, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>8 ms
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_getCurrentURL_byStatus_failed_FAIL2_exception"
                       class="testMethodDetails">
                     <pre style="padding: 5px; margin: 0;">java.lang.Throwable: Multiple failures (2):
                        
                        Failure 1 of 2:
                        java.lang.AssertionError: expected [URL] but found [https://www.google.co.in/]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                        	at org.testng.Assert.assertEquals(Assert.java:116)
                        	at org.testng.Assert.assertEquals(Assert.java:179)
                        	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                        	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                        	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:35)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                        
                        
                        Failure 2 of 2:
                        java.lang.AssertionError: Message----&gt; expected [true] but found [false]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertTrue(Assert.java:42)
                        	at utils.ErrorCollector.assertTrue(ErrorCollector.java:28)
                        	at utils.ErrorCollector.verifyTrue(ErrorCollector.java:65)
                        	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:36)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     </pre>
                  </div>
               </td>
            </tr>
            <tr id="testCasePack.Req_12345_init_byStatus_passed_PASS1_row"
                class="testMethodStatusPASS testMethodStatusCONF">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_init_byStatus_passed_PASS1_details')">init()</a>
               </td>
               <td nowrap="true">11:17:18</td>
               <td nowrap="true" align="right">4s
                  
               </td>
               <td nowrap="true">
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_init_byStatus_passed_PASS1_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>init
                     </div>
                     <div>
                        <b>Signature: </b>init()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:18
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>4s
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr></tr>
            <tr id="testCasePack.Req_12345_end_byStatus_passed_PASS2_row"
                class="testMethodStatusPASS testMethodStatusCONF">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_end_byStatus_passed_PASS2_details')">end()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">2s
                  
               </td>
               <td nowrap="true">
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_end_byStatus_passed_PASS2_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>end
                     </div>
                     <div>
                        <b>Signature: </b>end()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:25
                     </div>
                     <div>
                        <b>Duration: </b>2s
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr></tr>
         </table>
      </div>
      <div id="testMethodsByClass" style="display: none;">
         <table class="testMethodsTable" cellpadding="0" cellspacing="0">
            <tr>
               <td colspan="4">
                  <h3 style="display: inline;">testCasePack.Req_12345</h3>
               </td>
            </tr>
            <tr class="methodsTableHeader">
               <td width="100%">Name</td>
               <td nowrap="true">Started</td>
               <td nowrap="true">Duration</td>
               <td>Exception</td>
            </tr>
            <tr id="testCasePack.Req_12345_init_byClass_PASS1_row"
                class="testMethodStatusPASS testMethodStatusCONF">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_init_byClass_PASS1_details')">init()</a>
               </td>
               <td nowrap="true">11:17:18</td>
               <td nowrap="true" align="right">4s
                  
               </td>
               <td nowrap="true">
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_init_byClass_PASS1_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>init
                     </div>
                     <div>
                        <b>Signature: </b>init()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:18
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>4s
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr></tr>
            <tr id="testCasePack.Req_12345_verifyURL_byClass_FAIL2_row"
                class="testMethodStatusFAIL">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_verifyURL_byClass_FAIL2_details')">verifyURL()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">14 ms
                  
               </td>
               <td nowrap="true">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_verifyURL_byClass_FAIL2_exception')">java.lang.Throwable: 
                     Multiple failures (2):
                     
                     Failure 1 of 2:
                     java.lang.AssertionError: expected [Googe] but found [Google]
                     	at org.testng.Assert.fail(Assert.java:94)
                     	at org.testng.Assert.failNotEquals(Assert.java:513)
                     	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                     	at org.testng.Assert.assertEquals(Assert.java:116)
                     	at org.testng.Assert.assertEquals(Assert.java:179)
                     	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                     	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                     	at testCasePack.Req_12345.verifyURL(Req_12345.java:27)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                     	at java.lang.reflect.Method.invoke(Method.java:497)
                     	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                     	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                     	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                     	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                     	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                     	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                     	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                     	at org.testng.TestRunner.run(TestRunner.java:607)
                     	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                     	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                     	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                     	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                     	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                     	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                     	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                     	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                     	at org.testng.TestNG.runSuites(TestNG.java:1124)
                     	at org.testng.TestNG.run(TestNG.java:1096)
                     	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                     	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                     	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                     	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                     	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                     	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     
                     
                     Failure 2 of 2:
                     java.lang.AssertionError: 'This is Message'------&gt; expected [true] but found [false]
                     
                  </a>
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_verifyURL_byClass_FAIL2_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>verifyURL
                     </div>
                     <div>
                        <b>Description: </b>Verify Title
                     </div>
                     <div>
                        <b>Signature: </b>verifyURL()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>14 ms
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_verifyURL_byClass_FAIL2_exception"
                       class="testMethodDetails">
                     <pre style="padding: 5px; margin: 0;">java.lang.Throwable: Multiple failures (2):
                        
                        Failure 1 of 2:
                        java.lang.AssertionError: expected [Googe] but found [Google]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                        	at org.testng.Assert.assertEquals(Assert.java:116)
                        	at org.testng.Assert.assertEquals(Assert.java:179)
                        	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                        	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                        	at testCasePack.Req_12345.verifyURL(Req_12345.java:27)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                        
                        
                        Failure 2 of 2:
                        java.lang.AssertionError: 'This is Message'------&gt; expected [true] but found [false]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertTrue(Assert.java:42)
                        	at utils.ErrorCollector.assertTrue(ErrorCollector.java:28)
                        	at utils.ErrorCollector.verifyTrue(ErrorCollector.java:65)
                        	at testCasePack.Req_12345.verifyURL(Req_12345.java:28)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     </pre>
                  </div>
               </td>
            </tr>
            <tr id="testCasePack.Req_12345_getCurrentURL_byClass_FAIL3_row"
                class="testMethodStatusFAIL">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_getCurrentURL_byClass_FAIL3_details')">getCurrentURL()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">8 ms
                  
               </td>
               <td nowrap="true">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_getCurrentURL_byClass_FAIL3_exception')">java.lang.Throwable: 
                     Multiple failures (2):
                     
                     Failure 1 of 2:
                     java.lang.AssertionError: expected [URL] but found [https://www.google.co.in/]
                     	at org.testng.Assert.fail(Assert.java:94)
                     	at org.testng.Assert.failNotEquals(Assert.java:513)
                     	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                     	at org.testng.Assert.assertEquals(Assert.java:116)
                     	at org.testng.Assert.assertEquals(Assert.java:179)
                     	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                     	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                     	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:35)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                     	at java.lang.reflect.Method.invoke(Method.java:497)
                     	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                     	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                     	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                     	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                     	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                     	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                     	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                     	at org.testng.TestRunner.run(TestRunner.java:607)
                     	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                     	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                     	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                     	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                     	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                     	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                     	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                     	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                     	at org.testng.TestNG.runSuites(TestNG.java:1124)
                     	at org.testng.TestNG.run(TestNG.java:1096)
                     	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                     	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                     	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                     	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                     	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                     	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     
                     
                     Failure 2 of 2:
                     java.lang.AssertionError: Message----&gt; expected [true] but found [false]
                     
                  </a>
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_getCurrentURL_byClass_FAIL3_details"
                       class="testMethodDetails">
                     <div>
                        <b>Name: </b>getCurrentURL
                     </div>
                     <div>
                        <b>Description: </b>Verify URL
                     </div>
                     <div>
                        <b>Signature: </b>getCurrentURL()[pri:1, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:23
                     </div>
                     <div>
                        <b>Duration: </b>8 ms
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_getCurrentURL_byClass_FAIL3_exception"
                       class="testMethodDetails">
                     <pre style="padding: 5px; margin: 0;">java.lang.Throwable: Multiple failures (2):
                        
                        Failure 1 of 2:
                        java.lang.AssertionError: expected [URL] but found [https://www.google.co.in/]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
                        	at org.testng.Assert.assertEquals(Assert.java:116)
                        	at org.testng.Assert.assertEquals(Assert.java:179)
                        	at utils.ErrorCollector.assertEquals(ErrorCollector.java:44)
                        	at utils.ErrorCollector.verifyEquals(ErrorCollector.java:97)
                        	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:35)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                        
                        
                        Failure 2 of 2:
                        java.lang.AssertionError: Message----&gt; expected [true] but found [false]
                        	at org.testng.Assert.fail(Assert.java:94)
                        	at org.testng.Assert.failNotEquals(Assert.java:513)
                        	at org.testng.Assert.assertTrue(Assert.java:42)
                        	at utils.ErrorCollector.assertTrue(ErrorCollector.java:28)
                        	at utils.ErrorCollector.verifyTrue(ErrorCollector.java:65)
                        	at testCasePack.Req_12345.getCurrentURL(Req_12345.java:36)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                        	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                        	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                        	at java.lang.reflect.Method.invoke(Method.java:497)
                        	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
                        	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
                        	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
                        	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
                        	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
                        	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
                        	at org.testng.TestRunner.privateRun(TestRunner.java:753)
                        	at org.testng.TestRunner.run(TestRunner.java:607)
                        	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
                        	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
                        	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
                        	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
                        	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
                        	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
                        	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
                        	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
                        	at org.testng.TestNG.runSuites(TestNG.java:1124)
                        	at org.testng.TestNG.run(TestNG.java:1096)
                        	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
                        	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
                        	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
                        	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
                        	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
                        	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
                     </pre>
                  </div>
               </td>
            </tr>
            <tr id="testCasePack.Req_12345_end_byClass_PASS4_row"
                class="testMethodStatusPASS testMethodStatusCONF">
               <td width="100%" class="firstMethodCell">
                  <a onclick="toggleDetailsVisibility('testCasePack.Req_12345_end_byClass_PASS4_details')">end()</a>
               </td>
               <td nowrap="true">11:17:23</td>
               <td nowrap="true" align="right">2s
                  
               </td>
               <td nowrap="true">
                   
                  
               </td>
            </tr>
            <tr>
               <td colspan="4" class="detailsBox">
                  <div id="testCasePack.Req_12345_end_byClass_PASS4_details" class="testMethodDetails">
                     <div>
                        <b>Name: </b>end
                     </div>
                     <div>
                        <b>Signature: </b>end()[pri:0, instance:testCasePack.Req_12345@1b0375b3]
                     </div>
                     <div>
                        <b>Start time: </b>11:17:23
                     </div>
                     <div>
                        <b>End time: </b>11:17:25
                     </div>
                     <div>
                        <b>Duration: </b>2s
                        
                     </div>
                  </div>
               </td>
            </tr>
            <tr></tr>
         </table>
         <br></br>
      </div>
      <div style="margin-top: 15px; color: gray; text-align: center; font-size: 9px;">
         Generated with
         
         <a href="http://code.google.com/p/testng-xslt/" style="color: #8888aa;"
            target="_blank">
            TestNG XSLT
            
         </a>
      </div>
   </body>
</html>